scripts/ci/docker-compose.yml

```diff
@@ -18,61 +18,67 @@ version: "2.2"
 services:
   mysql:
     image: mysql:5.6
-    restart: always
+    restart: "no"
     environment:
       - MYSQL_ALLOW_EMPTY_PASSWORD=true
       - MYSQL_ROOT_HOST=%
     volumes:
       - ./mysql/conf.d:/etc/mysql/conf.d
-
   postgres:
     image: postgres:9.6
-    restart: always
+    restart: "no"
     environment:
       - POSTGRES_USER=postgres
       - POSTGRES_PASSWORD=airflow
       - POSTGRES_DB=airflow
-
   mongo:
     image: mongo:3
-    restart: always
+    restart: "no"
 
   cassandra:
     image: cassandra:3.0
-    restart: always
+    restart: "no"
 
   rabbitmq:
     image: rabbitmq:3.7
-    restart: always
+    restart: "no"
```

Comment:
`no` is the default right? Perhaps just remove the line instead?

Reply:
Yep. I think I left it as I wasn't really what will be the final version. I think "no" is better now. But that's something to be confirmed by @gerardo

Reply:
Yeah, we can switch it to no. It shouldn't affect the tests.

Reply:
OK. Cool. Thanks @gerardo. changing it.